frontend/components/service/addServiceForm.js

```diff
@@ -5,186 +5,180 @@ import Dialog from '@mui/material/Dialog';
-        <DialogTitle>
-          New Service
-          <IconButton
-            aria-label="close"
-            onClick={closeDialog}
-            size="medium"
-            sx={{
-              position: 'absolute',
-              right: 10,
-              top: 10,
-            }}
-          >
-            <Close />
-          </IconButton>
-        </DialogTitle>
-        <DialogContent>
-          <Grid container direction="column" alignItems="stretch">
-            <Grid item xs={12}>
-              <TextField
-                margin="normal"
-                fullWidth
-                required
-                id="name"
-                label="name"
-                value={name}
-                onChange={(event) => handleSetValue(event)}
-              />
-            </Grid>
-            <Grid item xs={12}>
-              <TextField
-                margin="normal"
-                fullWidth
-                required
-                id="code"
-                label="Service code"
-                value={barcode}
-                onChange={(event) => handleSetValue(event)}
-              />
-            </Grid>
-            <Grid item xs={12}>
-              {durationPriceList.map((element, index) => (
-                <DurationPrice
-                  key={index}
-                  index={index}
-                  item={element}
-                  durationPriceList={durationPriceList}
-                  handleRemoveDurationPrice={handleRemoveDurationPrice}
-                  reload={reload}
-                  setReload={setReload}
-                />
-              ))}
+    const handleAddEmployeeCheck = (val, employee) => {
+        if (val) {
+            setEmployeeCheckList([...employeeCheckList, employee]);
+        } else {
+            setEmployeeCheckList(
+                employeeCheckList.filter((emp) => emp.id != employee.id)
+            );
+        }
+    };
+
+    useEffect(() => {
+    }, [employeeCheckList, durationPriceList, reload]);
 
-              <Grid
-                container
-                direction="column"
-                justifyContent="center"
-                alignItems="center"
-              >
-                <Grid item={3}>
-                  <IconButton onClick={handleAddDurationPrice}>
-                    <AddIcon />
-                  </IconButton>
-                </Grid>
-              </Grid>
-            </Grid>
-            <Grid item xs={12}>
-              <TextField
-                margin="normal"
-                fullWidth
-                id="description"
-                label="Description"
-                multiline
-                rows={4}
-                value={description}
-                onChange={(event) => handleSetValue(event)}
-              />
-            </Grid>
-          </Grid>
 
-          {/*<div className="bg-success p-2 text-dark bg-opacity-10"  >New Service for 1H has been successfully added! </div>*/}
```

Comment:
clear this comment

Reply:
this is already cleared , (it might as deleted)